substrate/frame/vesting/src/lib.rs

```diff
@@ -784,3 +776,24 @@ where
 		Ok(())
 	}
 }
+
+impl<T: Config> VestedTransfer<T::AccountId> for Pallet<T>
+where
+	BalanceOf<T>: MaybeSerializeDeserialize + Debug,
+{
+	type Currency = T::Currency;
+	type Moment = BlockNumberFor<T>;
+
+	fn vested_transfer(
+		source: &T::AccountId,
+		target: &T::AccountId,
+		locked: BalanceOf<T>,
+		per_block: BalanceOf<T>,
+		starting_block: BlockNumberFor<T>,
+	) -> DispatchResult {
+		let schedule = VestingInfo::new(locked, per_block, starting_block);
+		let source = <T::Lookup as StaticLookup>::unlookup(source.clone());
+		let target = <T::Lookup as StaticLookup>::unlookup(target.clone());
```

Comment:
Inside `do_vested_transfer` you then call `lookup`. TLDR, `do_vested_transfer` should directly take `T::AccountId`

Reply:
done